view/adminhtml/templates/dashboard/signup.phtml

```diff
@@ -79,6 +78,20 @@ $state = $block->getPureclarityStateViewModel();
                             <input data-validate='{"required":true, "email": true}' type="text" name="email"
                                    id="pc-sign-up-email"/>
                         </p>
+                        <p class="pc-field">
+                            <label class="nr-label pc-tooltip-label" for="pc-sign-up-phone">
+                                <?= $block->escapeHtml(__('Phone Number')) ?>
+                            </label>
+                            <input type="text" name="phone" id="pc-sign-up-phone"/>
+                            <span class="pc-tooltip" id="pc-sign-up-phone-tooltip">?
+                                 <span class="pc-tooltip-text" id="pc-sign-up-phone-tooltip">
+                                <?= $block->escapeHtml(__(
+                                    'If you provide a phone number we will use it to contact you during your'
```

Comment:
For Shopify I've gone with the following - as we may use the phone number to contact them after the free trial to help with support issues.

If you provide a phone number we will use it to contact you during your free trial and for support. It will not be used for sales calls.

Reply:
fixed